app/views/subject/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="page_wrapper">
+	<%= render 'subject_form', locals: { title: "Edit Subject" }%>
```

Comment:
you don't need to use `locals`, you can use `render 'subject_form', title: "Edit Subject"` instead and in your `subject_form.html` template you can use simply `title` instead of `locals[:title]`